### DISCUSSION TITLE
I2C extension

### DISCUSSION THREAD

**Discussion by GiuseppeDiGuglielmo** [cocotb/cocotb#3514] [Q&A] [answered]
What is the status of the I2C extension?

I could find only
https://github.com/alexforencich/cocotbext-i2c

Wondering how much it has been tested or if there are alternatives.

Thanks,
G.

**Comment by ktbarrett** [comment, marked as answer]
There is no official I²C extension. If you want info about cocotbext-i2c you should ask @alexforencich or ask in an issue on that project's repo.

**Reply by GiuseppeDiGuglielmo** [reply]
Are there any plans for official support?

In any case, I will reach out @alexforencich

Thank you
